benchmarks/common/summarize_results.py

```diff
@@ -7,14 +7,29 @@ def create_summary(configurations: list[str],
                    solution_metrics: list[str],
                    solution_field_data: list[str],
                    benchmark: str,
+                   benchmark_uri: str,
                    summary_json: str) -> None:
     
+    """
+    Create a summary JSON file containing simulation results and metadata.
+        
+    Args:
+        configurations: List of configuration names
+        parameter_files: List of paths to parameter JSON files
+        mesh_files: List of paths to mesh files
+        solution_metrics: List of paths to metrics JSON files
+        solution_field_data: List of paths to solution field data files
+        benchmark: Name of the benchmark
+        benchmark_uri: URI of the benchmark
+        summary_json: Output path for the summary JSON file
+    """
+    
+    
     all_summaries = []
     for idx, config in enumerate(configurations):
-        print(idx, config)
         summary = {}
         summary["benchmark"] = benchmark
-        print(solution_metrics[idx])
+        summary["benchmark_uri"] = benchmark_uri
```

Comment:
can we do that similarly with the parameter files in one json that has the following information
- tool (as a string)
- tool_uri
- software version (x.y.z)
- benchmark (as a string)
- benchmark_uri (from MathmodDB)